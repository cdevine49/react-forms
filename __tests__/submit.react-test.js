import React from 'react';
import Submit from '../lib/components/submit';
import renderer from 'react-test-renderer';

describe("Submit", () => {
  test('renders with default props', () => {
    const tree = renderer.create(
      <Submit
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  test('accepts new props', () => {
    const tree = renderer.create(
      <Submit
        id="submitId"
        className="classy submit"
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});