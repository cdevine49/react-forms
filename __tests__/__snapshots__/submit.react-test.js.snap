// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Submit accepts new props 1`] = `
<input
  className="react-form-submit classy submit"
  id="submitId"
  type="submit"
  value="Submit"
/>
`;

exports[`Submit renders with default props 1`] = `
<input
  className="react-form-submit"
  type="submit"
  value="Submit"
/>
`;
